packages/react-server/core/__tests__/integration/containers/pages/SimpleContainerPage.js

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Yeah... once this lands I want to set up some tests for `<RootElement />` and `<RootContainer />`.